scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -15,6 +15,7 @@
   variables:
     RUSTFLAGS: "-D warnings"
   script:
+    - cargo install --locked --git https://github.com/paritytech/try-runtime-cli --branch liam-v0.1.0
```

Comment:
it'd be cool to publish it to crates.io and install from there, but it requires all its dependencies to be published...